README.md

```diff
@@ -19,19 +16,37 @@ access to their HTTP workloads in Kubernetes by easy integration offered via
 For DevOps and SRE teams, providing ingress for charms that support a relation
 to this charm will be possible via a simple `juju relate` command.
 
-## Project and community
+## Get started
+
+Refer to [Nginx Getting Started](https://charmhub.io/nginx-ingress-integrator/docs/getting-started) tutorial for more details.
```

Comment:
Nitpick: I would rephrase as `Refer to the [Nginx tutorial](https://charmhub.io/nginx-ingress-integrator/docs/getting-started) for more details on getting started.`